caddyconfig/httpcaddyfile/httptype.go

```diff
@@ -1282,19 +1282,24 @@ func matcherSetFromMatcherToken(
 	if tkn.Text == "*" {
 		// match all requests == no matchers, so nothing to do
 		return nil, true, nil
-	} else if strings.HasPrefix(tkn.Text, "/") {
-		// convenient way to specify a single path match
+	}
+
+	// convenient way to specify a single path match
+	if strings.HasPrefix(tkn.Text, "/") {
```

Comment:
I just don't like `else if` when all branches `return`. It can be simplified to simple `if` and the comment can be hoisted up above the `if`.